src/components/loading-indicators/loading-repository-list-item.component.js

```diff
@@ -52,21 +55,13 @@ export class LoadingRepositoryListItem extends Component {
 
   render() {
     return (
-      <View style={styles.container}>
-        <View style={styles.wrapper}>
-          <Animated.View
-            style={[styles.textBarTitle, { opacity: this.state.fadeAnimValue }]}
-          />
-
-          <Animated.View
-            style={[styles.textBarLine1, { opacity: this.state.fadeAnimValue }]}
-          />
-
-          <Animated.View
-            style={[styles.textBarLine2, { opacity: this.state.fadeAnimValue }]}
-          />
-        </View>
-      </View>
+      <Container>
+        <Wrapper>
+          <TextBarTitle style={{ opacity: this.state.fadeAnimValue }} />
+          <TextBarLine1 style={{ opacity: this.state.fadeAnimValue }} />
+          <TextBarLine2 style={{ opacity: this.state.fadeAnimValue }} />
```

Comment:
Setting opacity as props may be better, as mentioned in #838.

Reply:
@chinesedfan Why resolved this conversation?

Reply:
@chinesedfan got you, thank you for the explanation.